mne_bids/read.py

```diff
@@ -190,6 +208,14 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
         Full name of the data file
     bids_root : str
         Path to root of the BIDS folder
+    read_params : dict | None
+        Additional parameters for reading BIDS compatible data in a dictionary.
+        The keys of the dictionary must correspond with potential input
+        parameters of MNE-Python's read_raw_* function for the data format you
+        intend to read. For example, when reading fif data you can specify
+        `read_params` as {allow_maxshield=False}. Non-fitting parameters will
+        be ignored. Not all parameters are currently supported. If None,
```

Comment:
A dangerous route!

Reply:
+100. Don't silently ignore things.

Reply:
I am raising  `Userwarning` in verbose mode ... should I

1. always raise a warning?
1. always throw an error?
1. ... do something else? what?